pkg/cli/cvd.go

```diff
@@ -102,8 +111,8 @@ type CreateCVDOpts struct {
 	// Example: https://cs.android.com/android/platform/superproject/main/+/main:device/google/cuttlefish/host/cvd_test_configs/main_phone-main_watch.json;drc=b2e8f4f014abb7f9cb56c0ae199334aacb04542d
 	EnvConfig map[string]interface{}
 	// If true, perform the ADB connection automatically.
-	AutoConnect               bool
-	BuildAPICredentialsSource string
```

Comment:
let's continue using `BuildAPICredentialsSource` only using a valid filepath value to it. For example, the config parameter could have  special values like: `none`, `injected` and a file path value like: `/etc/cvdr/token.json`. Remember to update the commit message.